### PR TITLE
Support mobilenet_v1_1.0_224/Update schema for mobilenet/Enhance README.md for generating schema

### LINKED ISSUE #106: [MobileNet.js] operator type 43 is not supported by using new mobilenet_v1_1.0_224.tgz
Based on https://github.com/tensorflow/tensorflow/commit/28d43e5ada3c1e16b81c64b08cbbc273407a0347?short_path=7eb6a41#diff-7eb6a41d42575d4116fee4b816c60341 , all versions of mobilenet were updated to .tgz format from .zip with different contents:

```
├── mobilenet_v1_1.0_224_float_2017_11_08.zip
│   ├── labels.txt
│   └── mobilenet_v1_1.0_224.tflite

├── mobilenet_v1_1.0_224.tgz
│   ├── mobilenet_v1_1.0_224.ckpt.data-00000-of-00001
│   ├── mobilenet_v1_1.0_224.ckpt.index
│   ├── mobilenet_v1_1.0_224.ckpt.meta
│   ├── mobilenet_v1_1.0_224_eval.pbtxt
│   ├── mobilenet_v1_1.0_224_frozen.pb
│   ├── mobilenet_v1_1.0_224_info.txt
│   └── mobilenet_v1_1.0_224.tflite
```

It seems like the .ckpt.data-00000-of-00001, .ckpt.index and .ckpt.meta are saved by tf.train.Saver(), and there is no labels.txt file in the .tgz anymore.

Repro Steps:
1. Download http://download.tensorflow.org/models/mobilenet_v1_2018_02_22/mobilenet_v1_1.0_224.tgz
2. Upzip mobilenet_v1_1.0_224.tgz
3. Copy unzipped mobilenet_v1_1.0_224.tflite to ~/webml-polyfill/examples/mobilenet/model
4. keep 1.0 labels.txt from mobilenet_v1_1.0_224_float_2017_11_08.zip under ~/webml-polyfill/examples/mobilenet/model
5. npm start
6. Visit http://127.0.0.1:8081/examples/mobilenet/

Actual Result:
```
Uncaught (in promise) Error: operator type 43 is not supported.
    at MobileNet._addOpsAndParams (MobileNet.js:193)
    at MobileNet.createCompiledModel (MobileNet.js:36)
    at <anonymous>
```

There is no 43 in our operator types.

### PR DESCRIPTION
**Pull request by Wenzhao-Xiang**
Fix #106

### CODE REVIEW

**Comment by Wenzhao-Xiang** [comment]
@GreyZzzzzzXh  PTAL! Thanks!

**Comment by GreyZzzzzzXh** [comment]
Done, LGTM. @Wenzhao-Xiang

**Comment by huningxin** [comment]
Looks good. Let's merge it.
